wiki/models/__init__.py

```diff
@@ -1,6 +1,3 @@
-# -*- coding: utf-8 -*-
-from __future__ import absolute_import
-
 from django.conf import settings as django_settings
 from django.core.exceptions import ImproperlyConfigured
 from six import string_types, text_type
```

Comment:
Aren't these imports from `six` now unused?

Reply:
Removed the unused statement.